google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -988,6 +988,11 @@ public static BlobListOption currentDirectory() {
       return new BlobListOption(StorageRpc.Option.DELIMITER, true);
     }
 
+    /** Returns an option to set a delimiter '/'. */
+    public static BlobListOption delimiter() {
```

Comment:
Delimiter can be user defined, generally '/' is the one used most often, but I'd like to keep this open to other delimiters.

Reply:
done